firmware/src/pwm.h

```diff
@@ -9,6 +9,17 @@
 #include <avr/interrupt.h>
 #include <stdio.h>
 #include <util/delay.h>
+#include"conf.h"
+
+#ifdef MACHINE_ON
+#include "machine.h"
+#endif
+
+
+
+// // pwm macros
+#define set_pwm_duty_cycle(d)       OCR1A = d      //!< apply duty cycle 'd'
+#define set_pwm_off()               OCR1A = 0;     //!< d = 0

```

Comment:
NEVER use macros in lower case, this can be really confusing, please use functions for this case the compiler will optimize for you